tracker:
    pcd_path: /home/mike/Downloads/coke_csl.pcd
    z_max: 4.0
    z_min: 0.01
    downsampling_grid_size: 0.003
    KDLAdaptive:
        MaximumParticleNum: 1000
        Delta: 0.99
        Epsilon: 0.2
    ParticleFilter:
        LikelihoodThr: 0.00
        UseNormal: false
        ParticleNum: 600
        IterationNum: 1
    bin_size:
        x: 0.1
        y: 0.1
        z: 0.1
        roll: 0.1
        pitch: 0.1
        yaw: 0.1

